sbt/src/sbt-test/source-dependencies/macro-annotation/macros/src/main/scala/Macros.scala

```diff
@@ -3,24 +3,24 @@ import scala.language.experimental.macros
 import scala.annotation.StaticAnnotation
 
 object HelloMacro {
-  def impl(c: Context)(annottees: c.Tree*): c.Tree = {
+  def impl(c: Context)(annotatees: c.Tree*): c.Tree = {
```

Comment:
If "annottees" is the standard in macro lingo, let's just stick to it. I don't want to use sbt source as a platform of protest.